packages/consumption/src/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.ts

```diff
@@ -138,6 +138,37 @@ export class ReadAttributeRequestItemProcessor extends GenericRequestItemProcess
                         )
                     );
                 }
+
+                const queryForPendingRelationship: any = {
+                    "peer.address": foundLocalAttribute.shareInfo.peer.address,
+                    status: RelationshipStatus.Pending
+                };
+                const pendingRelationshipToPeer = await this.accountController.relationships.getRelationships(queryForPendingRelationship);
+
+                const queryForActivatedOrTerminatedRelationship: any = {
```

Comment:
If you are to keep this, think about replacing "activated" by "active". I think the emphasize should be on the fact that the Relationship is active now and not on the fact that it was activated in the past.